tests/server.py

```diff
@@ -739,9 +742,24 @@ def setup_test_homeserver(
             },
         }
     else:
+        if SQLITE_PERSIST_DB:
+            test_db_location = os.path.abspath("test.db")
+            logger.debug(
+                "Will persist db to %s",
+            )
+            # nuke the DB on disk if it already exists
```

Comment:
Will it ever exist? Doesn't the temp directory get fully destroyed before a run?

Reply:
My understanding: the directory gets destroyed before `trial` starts to run its tests, but not between test runs`; I expect we want each test to run on its own DB from scratch.

Reply:
Ah-ha! Right. 👍

Reply:
```suggestion
            # Ensure each test gets a clean database.
```